assets/javascripts/kitten/components/tooltips/static-tooltip.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow } from 'enzyme'
+import { StaticTooltip } from 'kitten/components/tooltips/static-tooltip'
+
+describe('<StaticTooltip />', () => {
+  describe('by default', () => {
+    const component = shallow(<StaticTooltip />)
+
+    it('a <StaticTooltip />', () => {
```

Comment:
Ici ce qu'on teste c'est plutôt que "ça a la classe k-StaticTooltip". Du coup je dirais `has the k-StaticTooltip class`.